dockstore-common/pom.xml

```diff
@@ -105,11 +105,194 @@
             <artifactId>jackson-annotations</artifactId>
         </dependency>
         <!-- broad - cromwell stuff -->
+
+        <dependency>
+            <groupId>org.broadinstitute</groupId>
+            <artifactId>wdl-draft3_2.12</artifactId>
+            <version>41</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>cats-effect_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.typelevel</groupId>
+                    <artifactId>cats-core_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
```

Comment:
This shouldn't be necessary
Guava should be in dependency management, so whatever version wdl-draft3_2.12 tries to bring in, dependency management should override it and avoid an enforcer warning.